pkg/network/tracer/tracer.go

```diff
@@ -31,6 +31,7 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/util/log"
 	"github.com/DataDog/ebpf"
 	"github.com/DataDog/ebpf/manager"
+	"github.com/pkg/errors"
```

Comment:
Shouldn't this be `errors`? It is part of stdlib now.

Reply:
Interesting, I didn't realize `goimports` pulled that one in instead. That is definitely not the one I was intending!